src/sql/azdata.proposed.d.ts

```diff
@@ -430,6 +430,17 @@ declare module 'azdata' {
 		enabled?: boolean;
 	}
 
+	export interface DropDownProperties {
+		/**
+		 * Adds a short hint that describes the expected value for the editable dropdown
+		 */
+		editableDropdownPlaceholder?: string;
```

Comment:
I think just `placeholder` would be better - and then the comment can indicate that it's only used for editable dropdowns.